projects/aws/upgrader/upgrade/kubeadm.go

```diff
@@ -293,3 +377,45 @@ func (u *InPlaceUpgrader) restoreCoreDNSConfig(ctx context.Context, cmpDir strin
 	logger.Info("restored coreDNS config successfully!")
 	return nil
 }
+
+func newTrue() *bool {
+	t := true
+	return &t
+}
+
+func generateKubeAdmConfCM(clusterConfig string) corev1.ConfigMap {
```

Comment:
nit: can you make this func return a pointer
```suggestion
func generateKubeAdmConfCM(clusterConfig string) *corev1.ConfigMap {
```